src/test/java/pl/wojciechkostecki/revel/controller/LocalControllerTest.java

```diff
@@ -0,0 +1,199 @@
+package pl.wojciechkostecki.revel.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.assertj.core.api.Assertions;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import pl.wojciechkostecki.revel.model.Local;
+import pl.wojciechkostecki.revel.repository.LocalRepository;
+
+import javax.transaction.Transactional;
+import java.time.LocalTime;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional
+class LocalControllerTest {
```

Comment:
```suggestion
class LocalControllerIT {
```
because we have integration tests here